### DISCUSSION TITLE
Comments

### DISCUSSION THREAD

**Discussion by jaschaio** [payloadcms/payload#798] [Q&A]
It seems like payload is not supporting comments out of the box, nor is there currently a plugin to enable it.

Has anybody done a basic comment system with payload?

e.G. a queue where comments need to be moderated before being published and can be threaded/nested.

**Comment by jmikrut** [comment]
Hey @jaschaio — yep, no one has had the need for comments yet, but this would be a great plugin to build. We probably won't be able to get to it for a bit yet as we have some other priorities on our radar, but if you end up building it, please share!
